lang/src/org/partiql/lang/typesystem/interfaces/operator/operators/UnaryOps.kt

```diff
@@ -26,3 +28,7 @@ abstract class UnaryOp internal constructor() : SqlOperator {
      */
     abstract fun invoke(sourceValue: ExprValue, typeParametersOfExpr: TypeParameters): ExprValue
 }
+
+abstract class NegOp : UnaryOp(OpAlias.NEG)
+abstract class NotOp : UnaryOp(OpAlias.NOT)
+abstract class PosOp : UnaryOp(OpAlias.POS)
```

Comment:
What is the reason for these being abstract?

Reply:
There are some methods not implemented yet in these classes. We need to inherit these classes to implement concrete operators. e.g. We can create a class `NegInt` to represent how `Neg` operator works on `Int` type.